pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -213,6 +213,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 				StatPrefix:       util.PassthroughCluster,
 				ClusterSpecifier: &tcp_proxy.TcpProxy_Cluster{Cluster: util.PassthroughCluster},
 			}
+			setAccessLog(env, node, tcpProxy)
```

Comment:
Should we add it for blackhole cluster as well?

Reply:
It’s not about the cluster. It’s about the listeners. And currently this is the only wildcard listener in the system. All others have access logs iirc. If that’s not the case, then yeah.

Reply:
There's also a fallback route, that might be used on ports that have other services, but that should be covered by the existing log config on the listener.